sympy/integrals/transforms.py

```diff
@@ -1799,7 +1799,7 @@ def hankel_transform(f, r, k, nu, **hints):
 
     >>> ht = hankel_transform(exp(-a*r), r, k, 0)
     >>> ht
-    a/(k**3*(a**2/k**2 + 1)**(3/2))
+    1/(a**2*(1 + k**2/a**2)**(3/2))
```

Comment:
Can anyone confirm these are the same? In both master and this branch, `noconds=False` gives the conditions `Eq(Abs(periodic_argument(polar_lift(k)**2, oo)), 0) & (Abs(periodic_argument(polar_lift(a)**2, oo)) < pi)`.